inc/namespace.php

```diff
@@ -29,6 +29,12 @@ function bootstrap() {
 		Block_Editor\bootstrap();
 	}
 
+	if ( $config['xmlrpc'] === false ) {
+		add_filter( 'xmlrpc_enabled', __NAMESPACE__ . '\\filter_xmlrpc_handler', 999 );
+		add_filter( 'xmlrpc_methods', __NAMESPACE__ . '\\filter_xmlrpc_methods_handler', 999 );
```

Comment:
Similar here too - there's a `__empty_array` callback provided with WP out of the box for this sort of scenario.